client/src/components/App.jsx

```diff
@@ -7,9 +7,13 @@ import React from 'react';
 
 import MainDisplay from './mainDisplay/MainDisplay.jsx';
 
+import photos from '../../../data/testPhotoData.js';
```

Comment:
In webpack how is your resolve set? It should allow for you to not have .js/.jsx